src/com/hms_networks/americas/sc/thingworx/config/TWConnectorConfig.java

```diff
@@ -116,6 +116,43 @@ public long getQueueDataPollIntervalMillis() throws JSONException {
     return queueDataPollIntervalMillis;
   }
 
+  /**
+   * Get the FTP user from the configuration.
+   *
+   * @return FTP user
+   * @throws JSONException if unable to get the FTP user from the configuration file
+   */
+  public String getFtpUser() throws JSONException {
+    String ftpUser;
+    if (configurationObject.has(TWConnectorConsts.CONNECTOR_CONFIG_FTP_USERNAME_KEY)) {
+      ftpUser = configurationObject.getString(TWConnectorConsts.CONNECTOR_CONFIG_FTP_USERNAME_KEY);
+    } else {
+      Logger.LOG_WARN("The ftp user was not set. The connector must be in UTC time.");
```

Comment:
I think these log messages should say "The Ewon must be set to the UTC time zone for proper connector functionality" or something along those lines. The connector itself is always in UTC, and the current message makes it sound like its possible to change that